pkg/jmxfetch/state.go

```diff
@@ -95,6 +96,6 @@ func StopJmxfetch() {
 }
 
 // InitRunner inits the runner and injects the dogstatsd server component
-func InitRunner(server dogstatsdServer.Component, logger jmxlogger.Component) {
-	state.runner.initRunner(server, logger)
+func InitRunner(server dogstatsdServer.Component, logger jmxlogger.Component, ipc ipc.Component) {
```

Comment:
Could you also please update the comment, since there are more components now than just dogstatsd?